### DISCUSSION TITLE
How to find what component causes installer to ask for restart?

### DISCUSSION THREAD

**Discussion by Muhomorik** [wixtoolset/issues#6808] [Questions]
Hello,

I have a problem with installation for the first time on user pc.
Installer shows the classic windows "Restart is required". 

I can figure out what component fails to register (COM and it doesn't work then). Everything works fine after restart.
There is another installer in the bundle, but they are not really related.

I've tried to create installation log, but I can't see anything suspicious in it.

Are there any specific lines I should be looking for in the log or any other way to figure out what makes installer to ask for restart?

/ Best regards
Dmitri

**Comment by robmen** [comment]
Look for words like "file in use". There should be something mentioned in the verbose log file.
